lib/cert/runner.rb

```diff
@@ -44,6 +41,25 @@ def run
       end
     end
 
+    # Command method for the :revoke_expired sub-command
+    def revoke_expired_certs!
+      login
```

Comment:
Should we also call `FastlaneCore::PrintTable.print_values(config: Cert.config, hide_keys: [:output_path], title: "Summary for cert #{Cert::VERSION}")` at this point?